rosbag2_transport/test/rosbag2_transport/test_record_all_use_sim_time.cpp

```diff
@@ -70,13 +77,15 @@ TEST_F(RecordIntegrationTestFixture, record_all_with_sim_time)
 
   size_t expected_messages = 10;
   auto ret = rosbag2_test_common::wait_until_shutdown(
-    std::chrono::seconds(2),
+    std::chrono::seconds(10),
```

Comment:
Changing from 2 to 10 seems a bit excessive, maybe 4-5 is enough to reduce the flakiness?

Reply:
@Blast545 I can change to 5.